### PR TITLE
feature : social Login

### PR DESCRIPTION
**Pull request by fixit2489**
Login feature

### CODE REVIEW

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+my second file
```
Comment (ghost) [inline file2.txt]: Please capitalize the code

**Review by ghost — CHANGES_REQUESTED** [review]
please make the changes and push

File: file2.txt (hunk 1 above)
Comment (fixit2489) [reply file2.txt]: Completed the changes, please check

**Comment by ghost** [comment]
Everything looks good

**Review by ghost — APPROVED** [review]

**Pull request merged**